sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,14 +1,19 @@
 # Release History
 
-## 1.2.4 (Unreleased)
+## 2.0.0-beta.1 (Unreleased)
 
-## 1.2.4-beta.2 (Unreleased)
+This release continues with the changes from `1.2.4-beta.1`.
```

Comment:
This will also have the bug fixes that went out in 1.2.4 right? 

```suggestion
This release continues with the changes from `1.2.4` and `1.2.4-beta.1`.
```